### ISSUE TITLE
Visibility bug for grass not visible at start

### ISSUE THREAD

**Issue by EmiOnGit** [EmiOnGit/warblersneeds#16]
## Describe the bug
If an entity is spawned with `ComputedVisibility` set to not visible, 
it will not registered and be invisible even if `ComputedVisibility` is set to visible.

## To Reproduce
It can be seen in the load_many_chunks example.
100 chunks should be loaded into a grid but only the ones that are visible in the default view are registered.
Use `cargo -r r --example load_many_chunks`
to reproduce.
Use keys Q,W,E,A,S,D for movement

**Comment by EmiOnGit** [comment]
Any idea @janhohenheim ?

**Comment by janhohenheim** [comment]
Huh, strange. I'll check later

**Comment by janhohenheim** [comment]
I tried replacing `Changed<Grass>` by `Changed<ComputedVisibility>`, but turns out `ComputedVisibility` is mutably dereferenced and thus "changed" every frame. The best I can think of for now is removing the `Changed` constraint, querying `ChangeTrackers` and adding the following to the loop for a tiny optimization:
```rust
if !tracker.is_changed() && entity_cache.contains(&entity) {
    continue;
}
```
Quite suboptimal, because we'd iterate over all entities :/

**Comment by EmiOnGit** [comment]
Glad we came up to the same conclusion..

**Comment by EmiOnGit** [comment]
Should be solved by #19
